stack.go

```diff
@@ -27,21 +27,17 @@ type Frame struct {
 	Line int `json:"lineno"`
 }
 
-// A Stack is a slice of frames
-type Stack []Frame
+// A stack is a slice of frames
+type stack []frame
 
-// BuildStack uses the Go runtime to construct a slice of frames optionally skipping the number of
-// frames specified by the input skip argument.
-func BuildStack(skip int) Stack {
-	stack := make(Stack, 0)
+// buildStack converts []runtime.Frame into a JSON serializable slice of frames
+// optionally skipping the number of frames specified by the input skip argument.
```

Comment:
You can get rid of the "optionally skipping ..." as we are not taking a skip argument anymore.